job_templates/ci_job.xml.em

```diff
@@ -225,7 +225,8 @@ echo "# END SECTION"
 @[  end if]@
 @[elif os_name == 'windows']@
 setlocal enableDelayedExpansion
-rmdir /S /Q ws workspace "work space"
+REM rmdir /S /Q ws workspace "work space"
+rmdir /S /Q ws/build ws/install ws/source ws/*.repos ws/env.bat
```

Comment:
`ws/source` -> `ws/src`, at least that's what I see in `ci_windows/ws` on icecube

Reply:
:+1: Fixed.